CONTRIBUTING.md

```diff
@@ -0,0 +1,195 @@
+
+We welcome many different types of contributions including:
+
+* New features
+* Builds, CI/CD
+* Bug fixes
+* Documentation
+* Issue Triage
+* Answering questions on Slack/Mailing List/Matrix
+* Web design, helping to maintain the Flatcar website
+* Communications / Social Media / Blog Posts
+* Release management
+
+Not everything happens through a GitHub pull request. Please come to our
+[meetings](### Monthly Office hours and Developer Syncs) or [contact us](maintainers-noreply@flatcar-linux.org) and let's discuss how we can work
+together. 
+
+## Participate and contribute
+
+If you are thinking of making a contribution, then please engage with the project as early as possible -- by commenting on an existing issue, or creating a new issue, on GitHub. Consider the project’s mission, and how your contribution furthers it.
+Making your intent visible early on can be a major factor for getting your work accepted.
+
+For an introduction to the Flatcar SDK and a walk-through of common developer cases like customising the OS image (e.g. adding or upgrading packages), have a look at our [developer guides](https://www.flatcar.org/docs/latest/reference/developer-guides/); particularly the [howto on building custom images from source](https://www.flatcar.org/docs/latest/reference/developer-guides/sdk-modifying-flatcar/).
+The guides aim to provide a solid base for working with the SDK to help you filing successful PRs to the Flatcar project.
+
+For the general guidelines on making PRs/commits easier to review, please check out the project's [contribution guidelines on git](contributions-git.md).
+
+
+### Monthly Office hours and Developer Syncs
+
+We maintain a [Google Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com) ([iCal](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)) with both our Office Hours and Developer Sync meeting series which interested folks can comfortably import into the calendar app of their choice.
+
+Join us in our monthly [office hours meetings](../../discussions/categories/office-hours-agenda) to engage with the Flatcar User community interactively, to learn about the project's directions, and to discuss contributions. We also conduct the occasional user-focused demo of technologies related to image-based Linux.
+Lastly, the call includes a brief Release Planning with an update on the changes in the next immediate releases.
+
+If you'd like to share something or if you have a pressing issue you'd like discussed, please let us know.
+Either comment on the respective meeting discussion, reach out to us on Matrix (see below), or simply join the meeting and speak up in the meeting's Q&A.
+
+**Flatcar Office Hours are on the second Tuesday of every month at 9pm IST / 5:30pm CEST / 3:30pm GMT / 11:30am EDT / 8:30am PST**
```

Comment:
Not a new issue. In the past it was discussed a few times.
Anyway In my opinion, it is not a good idea to list all possible timezone entries for a meeting time like  that. When the next winter time starts in Oct., we have to fix all the entries in this doc again, and repeat the same every year in Spring and Autumn. Same story for the Flatcar website. Huge maintenance burden.
That's why I always suggest writing only UTC(GMT).